app/org/sagebionetworks/bridge/models/schedules/PersistentActivityScheduler.java

```diff
@@ -29,7 +29,8 @@
             DateTime scheduledTime = getFirstEventDateTime(context, finishedId+"," + schedule.getEventId());
 
             if (scheduledTime != null) {
-                DateTime localDateTime = scheduledTime.withZone(context.getInitialTimeZone());
+                DateTime localDateTime = scheduledTime.withZone(context.getStartsOn().getZone());
```

Comment:
I will revert this. I don't want to change this... instead I decided just to mark persistent tasks as available, even if the user moves timezones very near to the event timestamp causing the task to shift to scheduled. I've split the endsOn and initial time zones in the persistent activities test.